e2echart/e2e-chart-template.html

```diff
@@ -223,6 +230,9 @@ <h5 class="modal-title">Resource</h5>
                 return [item.locator, ` (kubelet container readiness)`, "ContainerReadinessErrored"];
             }
         }
+        if (m && isKubeletStartupProbeFailure(item)){
+            return [item.locator, ` (kubelet container readiness)`, "StartupProbeFailed"];
```

Comment:
Should the message be named startup instead of readiness since it is a startup probe?

Reply:
I left it as `kubelet container readiness` because the StartupProbeFailed is still part of container readiness and the event came from reading the kubelet log.

Reply:
There are three different probes with kubelet: liveness, readiness and startup. I assume you want to use a separate startup instead of readiness to indicate the kind of probe this is about.

Reply:
you are right -- thanks!